### PR TITLE
update typograph sizes per spec

### PR DESCRIPTION
**Pull request by Cheshire89**
typography edits didnt go through for some reason
https://rocketcom.atlassian.net/browse/ASTRO-1444?atlOrigin=eyJpIjoiZWUyYmQwNjFjZDI2NDIwY2I0OTg4YTNhZGFmYmM1MWIiLCJwIjoiaiJ9

### CODE REVIEW

**Comment by Cheshire89** [comment]
@avillaindesire your not in the reviewers list so i'm trying to ping you this way

**Review by dmcalester — APPROVED** [review]

File: src/scss/common/_typography.scss [hunk 1]
```diff
@@ -2,24 +2,25 @@ h1,
 h2,
 h3 {
 	font-family: var(--fontFamily);
-  font-weight: 300;
+  font-weight: 400;
```
Comment (amalyah) [inline src/scss/common/_typography.scss:5]: Suggest removing non-shared font-weight (400) from this shared declaration.

File: static/css/astro.core.css [hunk 2]
```diff
@@ -648,19 +648,20 @@ li.light-theme {
...
   --primaryLightHover: #92cbff4D;
   /* TODO: this is a temporary fix, the use of opacity from Sketch is new and not accounted for in CSS */
-  --primaryDark: #3a87cf;
+  --primaryDark: var(--colorSecondaryLighten2, #3a87cf);
   --primaryDarkHover: #6b8ca5;
-  --primaryElementText: #080c11;
-  --inputBackground: #FFFFFF;
+  --primaryElementText: var(--colorTertiaryDarken4, #080c11);
+  --inputBackground: var(--colorWhite, #FFFFFF);
   --inputDark: #080c11;
```
Comment (amalyah) [inline static/css/astro.core.css:665]: This should also be --colorTertiaryDarken4.

File: static/css/astro.core.css [hunk 3]
```diff
@@ -1023,21 +1024,22 @@ h2,
 h3 {
   font-family: "Roboto", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
   font-family: var(--fontFamily);
-  font-weight: 300;
+  font-weight: 400;
```
Comment (amalyah) [inline static/css/astro.core.css:1027]: Suggest removing non-shared font-weight (400) from this shared declaration.

File: static/css/astro.core.css [hunk 4]
```diff
@@ -648,19 +648,20 @@ li.light-theme {
...
   --surfaceElements: #1b2d3e;
-  --primary: #4dacff;
-  --primaryLight: #92cbff;
+  /* TODO: this is an unofficial Astro color, but a required KM color */
+  --primary: var(--colorSecondary, #4dacff);
+  --primaryLight: var(--colorSecondaryLighten2, #92cbff);
   --primaryLightHover: #92cbff4D;
   /* TODO: this is a temporary fix, the use of opacity from Sketch is new and not accounted for in CSS */
-  --primaryDark: #3a87cf;
+  --primaryDark: var(--colorSecondaryLighten2, #3a87cf);
```
Comment (amalyah) [inline static/css/astro.core.css:661]: This should be --colorSecondaryDarken1

File: static/css/astro.core.css (hunk 3 above)
Comment (dmcalester) [reply static/css/astro.core.css:1027]: Compiled CSS

File: src/scss/common/_typography.scss (hunk 1 above)
Comment (Cheshire89) [reply src/scss/common/_typography.scss:5]: It is being over written for the h3 tag lower in the code
however h1 and h2 are 400 weight

Comment (amalyah) [reply src/scss/common/_typography.scss:5]: Is it worth having it here and not just in the individual heading definitions?

Comment (Cheshire89) [reply src/scss/common/_typography.scss:5]: yes it is accepted css practice

File: static/css/astro.core.css (hunk 2 above)
Comment (Cheshire89) [reply static/css/astro.core.css:665]: computed css files should not be reviewed i.e anything that starts with `astro`

 As they are triggered by compiler which might be affected many different things such as code formatters, variable formatters (which generate our RGB values) for obvious reasons we either have limited control over output or tweaks will lead to global changes

**Review by Full-lifey — APPROVED** [review]

**Review by dmcalester — APPROVED** [review]

**Pull request merged**